### ISSUE TITLE
Error message is not parsed properly

### ISSUE THREAD

**Issue by nav16** [mirego/activerecord_json_validator#38]
I am using it in a engine `core`, on `Rails 5.2.0`

    `validates :children, json: { schema: CHILDREN_SCHEMA }`

I am not getting proper error message

```
"children": [
            "translation missing: en.activerecord.errors.models.core/user.attributes.children.invalid_json"
        ]
```
Any suggestions will be helpful.

**Comment by nav16** [comment]
I think customising the error is not possible as the message is generated by `json-schema `
[json-schema/required](https://github.com/ruby-json-schema/json-schema/blob/6333f0ca18563c613cca0ae994bbf69fe58a9421/lib/json-schema/attributes/required.rb#L21)

**Comment by remiprev** [comment]
Yes you’re right, the default error message is `invalid_json`, which is then translated by ActiveRecord.

The way you’re using the `message` option is correct — you could pass the `errors` argument through a custom method to parse errors into a structured hash:

```ruby
class Foo < ApplicationRecord
  validates :children, json: { message: ->(errors) { Foo.parse_json_errors(errors) }, schema: CHILDREN_SCHEMA }

  def self.parse_json_errors(errors)
    # Here, you can parse the `errors` array into something more meaningful
  end
end
```

I hope this helps you!
